src/test/java/com/artipie/asto/google/GoogleApiHandlerOnGetRequest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.artipie.asto.google;
+
+import io.vertx.core.Handler;
+import io.vertx.reactivex.core.http.HttpServerRequest;
+
+/**
+ * GoogleApiHandlerOnGetRequest.
+ *
+ * @since 0.18
+ */
+public final class GoogleApiHandlerOnGetRequest implements Handler<HttpServerRequest> {
```

Comment:
@Guseyn is this a class used only for testing purposes? A mock respone? If yes, we should move it to a test package and maybe use more compact name. The class can be used for every request as it looks like, not just `GET`

Reply:
@Vatavuk It's in the test package. This is only for `GET` requests, for other request we would have json reesponse

Reply:
@Guseyn o I see, my bad.